airframe-rx-html/src/main/scala/wvlet/airframe/rx/html/Attrs.scala

```diff
@@ -527,98 +527,98 @@ trait MouseEventAttrs {
+  lazy val ondblclick = handler[compat.MouseEvent]("ondblclick")
 
   /**
     * Script to be run when an element is dragged
     */
-  val ondrag = attr("ondrag")
+  val ondrag = handler[compat.MouseEvent]("ondrag")
 
   /**
     * Script to be run at the end of a drag operation
     */
-  lazy val ondragend = attr("ondragend")
+  lazy val ondragend = handler[compat.MouseEvent]("ondragend")
 
   /**
     * Script to be run when an element has been dragged to a valid drop target
     */
-  lazy val ondragenter = attr("ondragenter")
+  lazy val ondragenter = handler[compat.MouseEvent]("ondragenter")
 
   /**
     * Script to be run when an element leaves a valid drop target
     */
-  lazy val ondragleave = attr("ondragleave")
+  lazy val ondragleave = handler[compat.MouseEvent]("ondragleave")
 
   /**
     * Script to be run when an element is being dragged over a valid drop target
     */
-  lazy val ondragover = attr("ondragover")
+  lazy val ondragover = handler[compat.MouseEvent]("ondragover")
 
   /**
     * Script to be run at the start of a drag operation
     */
-  lazy val ondragstart = attr("ondragstart")
+  lazy val ondragstart = handler[compat.MouseEvent]("ondragstart")
 
   /**
     * Script to be run when dragged element is being dropped
     */
-  lazy val ondrop = attr("ondrop")
+  lazy val ondrop = handler[compat.MouseEvent]("ondrop")
 
   /**
     * The mousedown event is raised when the user presses the mouse button.
     *
     * MDN
     */
-  lazy val onmousedown = attr("onmousedown")
+  lazy val onmousedown = handler[compat.MouseEvent]("onmousedown")
 
   /**
     * The mousemove event is raised when the user moves the mouse.
     *
     * MDN
     */
-  lazy val onmousemove = attr("onmousemove")
+  lazy val onmousemove = handler[compat.MouseEvent]("onmousemove")
 
   /**
     * The mouseout event is raised when the mouse leaves an element (e.g, when the mouse moves off of an image in the
     * web page, the mouseout event is raised for that image element).
     *
     * MDN
     */
-  lazy val onmouseout = attr("onmouseout")
+  lazy val onmouseout = handler[compat.MouseEvent]("onmouseout")
 
   /**
     * The mouseover event is raised when the user moves the mouse over a particular element.
     *
     * MDN
     */
-  lazy val onmouseover = attr("onmouseover")
+  lazy val onmouseover = handler[compat.MouseEvent]("onmouseover")
 
   /**
     * The mouseup event is raised when the user releases the mouse button.
     *
     * MDN
     */
-  lazy val onmouseup = attr("onmouseup")
+  lazy val onmouseup = handler[compat.MouseEvent]("onmouseup")
 
   /**
     * Specifies the function to be called when the window is scrolled.
     *
     * MDN
     */
-  lazy val onscroll = attr("onscroll")
+  lazy val onscroll = handler[compat.UIEvent]("onscroll")
 
   /**
     * Fires when the mouse wheel rolls up or down over an element
     */
-  lazy val onwheel = attr("onwheel")
+  lazy val onwheel = handler[compat.WheelEvent]("onwheel")
```

Comment:
Note)
Note) onscroll receives `WheelEvent`, not `MouseEvent`
https://developer.mozilla.org/en-US/docs/Web/API/Element/wheel_event
https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onwheel